src/theme.tsx

```diff
@@ -1763,6 +1763,17 @@ export const theme = createTheme({
 				},
 			},
 		},
+		MuiSnackbar: {
+			styleOverrides: {
+				anchorOriginBottomRight: ({ theme }) => ({
+					[theme.breakpoints.up('sm')]: {
+						// Make room for the 'need help' button
+						// TODO add a theme.spacing entry between 32 and 64
```

Comment:
Do we have more cases that would benefit from that? Aside from it being a breaking change (requiring incrementing all spacings above it by 1) I'm also hesitant to add things just for a special case

Reply:
not that I can think of right now @myarmolinsky but the leap from 32px to 64px is pretty big so I expect it will come in handy in the future

Reply:
here's the default MUI spacing grid: https://m2.material.io/design/layout/understanding-layout.html#material-measurements

each step is a 8px increment, it's not exponential in any way.

Reply:
Yes of course! I was just commenting to raise the discussion